src/hb-ot-cff1-table.cc

```diff
@@ -125,6 +125,67 @@ static const uint8_t standard_encoding_to_sid [] =
     0,   144,   0,    0,    0,  145,    0,    0,  146,  147,  148,  149,    0,    0,    0,    0
 };
 
+/* SID to standard string */
+static const char *sid_to_standard_string [] =
```

Comment:
We need better storage for this.  See `hb-string-array.hh`.

Reply:
ok